%-------------------------
% Resume in Latex
% Author : Jared Galloway
%------------------------

\documentclass[letterpaper,9pt]{article}

\usepackage{latexsym}
\usepackage[empty]{fullpage}
\usepackage{titlesec}
\usepackage{marvosym}
\usepackage[usenames,dvipsnames]{color}
\usepackage{verbatim}
\usepackage{enumitem}
\usepackage[hidelinks]{hyperref}
\usepackage{fancyhdr}
\usepackage[english]{babel}
\usepackage{fontawesome5}

\usepackage{multibib}
\newcites{prim}{first/co-first author publications}
\newcites{supp}{supporting author publications}

\pagestyle{fancy}
\fancyhf{} % clear all header and footer fields
\fancyfoot{}
\renewcommand{\headrulewidth}{0pt}
\renewcommand{\footrulewidth}{0pt}

% Adjust margins
\addtolength{\oddsidemargin}{-0.5in}
\addtolength{\evensidemargin}{-0.5in}
\addtolength{\textwidth}{1in}
\addtolength{\topmargin}{-.5in}
\addtolength{\textheight}{1.0in}

\usepackage{xcolor}
\definecolor{mylightblue}{RGB}{70, 130, 180}
\definecolor{mygrey}{RGB}{124, 127, 134}
\usepackage{hyperref}
\hypersetup{
    colorlinks=true,
    linkcolor=mygrey,
    citecolor=black,
    urlcolor=mylightblue,
}
\urlstyle{same}
\let\oldhref\href
% \renewcommand{\href}[2]{\oldhref{#1}{\bfseries#2}}

\raggedbottom
\raggedright
\setlength{\tabcolsep}{0in}

% Sections formatting
\titleformat{\section}{
  \vspace{-4pt}\scshape\raggedright\large
}{}{0em}{}[\color{black}\titlerule \vspace{-5pt}]

%-------------------------
% Custom commands

% make a minipage rule for personal statement
\newcommand{\personalstatement}[1]{
  \begin{center}
  \begin{minipage}[t]{0.96\textwidth}
    \linespread{0.90}\selectfont
    \vspace{.1cm}\small{
      #1 \vspace{1pt}
    }
  \end{minipage}
  \end{center}
}

\newcommand{\resumeNoWrapItem}[2]{
  \item\small{
    \textbf{#1}{: #2 \vspace{-2pt}}
  }
}

\newcommand{\resumeItem}[2]{
  \begin{minipage}[t]{0.9\textwidth}
    \linespread{0.80}\selectfont
    \vspace{.1cm}\item\small{
      \textbf{#1}{: #2 \vspace{1pt}}
    }
  \end{minipage}
}

\newcommand{\resumeSubheading}[4]{
  \vspace{-1pt}\item
    \begin{tabular*}{0.97\textwidth}[t]{l@{\extracolsep{\fill}}r}
      \textbf{#1} & #2 \\
      \textit{\small#3} & \textit{\small #4} \\
    \end{tabular*}\vspace{-3pt}
}

\newcommand{\resumeSoftwareSubheading}[3]{
  \vspace{-3pt}\item
    \begin{tabular*}{0.97\textwidth}[t]{l@{\extracolsep{\fill}}r}
      \textbf{#1}\label{#1} & #2 \\
    \end{tabular*}\vspace{-7pt}
    \begin{center}
    \begin{minipage}{0.87\textwidth}
    \linespread{0.80}\selectfont
        {\small#3}
    \end{minipage}\vspace{-7pt}
    \end{center}
}


\newcommand{\resumeSubItem}[2]{\resumeNoWrapItem{#1}{#2}\vspace{-2pt}}

\renewcommand{\labelitemi}{$\circ$}
\renewcommand{\labelitemii}{$\cdot$}

\newcommand{\resumeSubHeadingListStart}{\begin{itemize}[leftmargin=*]}
\newcommand{\resumeSubHeadingListEnd}{\end{itemize}}
\newcommand{\resumeItemListStart}{\begin{itemize}[]}
\newcommand{\resumeItemListEnd}{\end{itemize}\vspace{-5pt}}

%-------------------------------------------
%%%%%%  CV STARTS HERE  %%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{document}

\begin{tabular*}{\textwidth}{l@{\extracolsep{\fill}}r}
  \textbf{\Large Jared Galloway} \\
  \textit{Curriculum vitae} \\
  &  jaredgalloway07@gmail.com  \\
  &  +1-406-579-6768 \\
  &  \href{https://github.com/jgallowa07}{\faGithub} \
  \href{https://www.linkedin.com/in/jaredgalloway/}{\faLinkedin} \
  \href{https://orcid.org/0000-0001-5838-7840}{\faOrcid} \ 
  \href{https://scholar.google.com/citations?user=zWo6lFcAAAAJ&hl=en}{\faGraduationCap} \
  \href{http://jaredgalloway.org/}{\faGlobe}
\end{tabular*}

%-----------PERSONAL STATEMENT--------

\section{Personal Statement}

\personalstatement{

I am a Data Scientist with a
computer and information science background,
a focus on bioinformatics.
I have seven years of experience in an academic research environment 
conducting various flavors of 
methods development, implementation, and downstream analysis
in the biological sciences.
This has provided me with a strong intuition
for taking noisy empirical data through the various steps 
of filtering artifacts, normalization,  modeling, and visualization
to produce tidy, coherent, and reproducible datasets.
My education in product design is often reflected in my work
as I find particular interest in creating intuitive and aesthetically
pleasing data visualization and software interfaces. 
I take pride in clearly communicating my work
and very much enjoy the dicussion that follows.

% Being raised in Montaave been fortunate enough to
% experience and foster appreciation for our natural world.
% This is what drew me to biological sciences and also plays out
% into much of my personal life as an avid skier, biker, and hiker.
% I find that my passion  has been a driving force
% in my career as a data scientist, as I am often drawn to projects
% that have a direct impact on the environment, or human health.

% While my career has primarily rewarded insight and results 
% that inform our understanding of biological mechanisms, 
% I find that I'm inclined towards the design and engineering
% behind the research.
% In my life outside work I find real appreciation and and in inclination towards
% eloquent design of a product that feels natural to the end-user.

%I enjoy playing the game of detective with these datasets by
%visualizing, hypothesizing, and then leveraging the appropriate 
%statistical tests of significance.
% I am a strong communicator, and have a knack for teaching, 
% I have become skilled at writing reproducible data-processing workflows 
% which tie publicly available software 
% with "in-house" software written either by myself,
% or in collaborate efforts with students, post-Docs, and faculty.
% I'm fascinated by both the mechanisms that drive our world, and the way
% in which we can engineer solutions to problems that arise from these mechanisms.
% While a good chunk of the methods being performed in the pipeline are often heavily
% informed by either previous studies, or a math-inclined collegue, 
% I find that the process of implementing or retro-fitting these methods 
% has provided me with a strong intuition for the underlying mathematics and statistics.
% This can be seen in the methods I've  

}

%-----------EDUCATION-----------------
\section{Education}
    \resumeSubHeadingListStart
        \resumeSubheading
            {Master of Science, Biology}{University of Oregon}
            {Knight Campus - Bioinformatics and Genomics Program}{June 2019 -- December 2020, Eugene, OR}
        \resumeSubheading
            {Bachelor of Science, Computer \& Information Science}{University of Oregon}
            {College of Arts and Sciences - Departmental Honors - Minor in Product Design}{Aug 2013 -- July 2018, Eugene, OR}
    \resumeSubHeadingListEnd

%--------PROGRAMMING SKILLS------------
\section{Technical Skills}

\resumeSubHeadingListStart
    \resumeSubItem{Languages}
        {
            Python, R, C/C++, Bash, Java, Javascript
        }
    \resumeSubItem{Mathematics}
        {
            Advanced Statistics and Bayesian Modeling, 
            Linear Algebra, 
            Discrete Math, 
            Calculus
        }
    \resumeSubItem{Operating systems}
        {
            Linux, OSX, Windows
        }
    \resumeSubItem{Software}
        {
            Git, GitHub Actions, Travis CI, Quay % version control 
            GNU make, cmake, scons, % build sysntems
            zshell/bash, vim, tmux, % command line interface 
            VSCode,
            \LaTeX, html, markdown,
            SLURM-based HPC clusters, 
            AWS S3, % AWS
            Conda/Mamba, % dependency management 
            Docker, Apptainer (formerly Singularity) % containerization 
            JAX, Tensorflow/Keras, PyTorch, 
            Pandas, Polars, Numpy, Xarray, 
            Matplotlib, Plotnine (grammar of graphics), Altair (interactive visualization),
            Scikit-learn, SciPy, BioPython,
            Nextflow, Snakemake
        }
\resumeSubHeadingListEnd

%-----------EXPERIENCE-----------------
    
    % Develop
    % Design
    % Drive
    % Evaluate
    % Implement
    % Manage
    % Build
    % Organize
    % Demonstrate
    % Reported
    % Visualize
    % Analyze X
    % Utilize
    % Leverage
    % Collaborate
    % Construct
    % Assist
    % Conduct
    % Lead / led

\section{Experience}
    \resumeSubHeadingListStart

        \resumeSubheading
            {Bioinformatics Analyst II}
            {Fred Hutchinson Cancer Center}
            {Public Health Sciences Division - Matsen Lab}
            {Jan 2021 - Current, Bozeman, MT (Remote)}
            \resumeItemListStart

                \resumeItem{Modeling Deep Mutational Scanning datasets}
                    {
                        Constructed a novel approach for jointly modeling 
                        multiple distinct DMS experimental datasets.
                        the model was validated via simulation, and integrated into
                        a JAX-based python package, \href{https://matsengrp.github.io/multidms/}{multidms}.
                        Application of this method on empirical data has resulted the identification of 
                        specific shifts in mutational effect between 
                        three separate homologs of the SARS-CoV-2 spike protein \cite{Haddox_2023}.
                    }

                \resumeItem{Algorithmic design for mutational fitness estimation}
                    {
                        Implemented complex algorithm for 
                        estimating mutational fitness trajectories (through time) for 
                        every possible codon-mutation that may have occurred 
                        within a phylogeny (evolutionary history) of $>13$ million empirical viral genomes.
                        Using the Snakemake workflow manager, I integrated this
                        with various other upstream quality control filtering, 
                        and downstream analysis to produce the results driving an ongoing study.
                    }

                \resumeItem{Genomic surveillance of viral escape}
                    {
                        Built and deployed custom, public Nextstrain pages for viruses such as 
                        \href{https://nextstrain.org/groups/dms-phenotype/ncov/Dadonaite-2023-Cell}{SARS-CoV-2},
                        Influenza, Lassa, and others 
                        -- each of which provide estimates of viral escape to antibodies of interest.
                        This work ultimately has provided analysis tools for various studies,
                        inform the potential effectiveness of these antibodies as therapeutics.
                    }

                \resumeItem{Bioinformatics for studying affinity maturation in germinal centers}
                    {
                        Created an efficient Nextflow pipeline 
                        which leveraged high performance computing clusters and
                        software containerization to produce $>100$ feature-rich phylogenetic trees 
                        of B-cell lineages within germinal centers of engineered mice.
                        This work has provided the foundational results for an ongoing large-scale, 
                        collaborative study.
                    }

                \resumeItem{Administrative data management}
                    {
                        Contributed to the documentation and execution of best-practice
                        management and storage of sensitive lab data in various computational 
                        environments including HPC file systems and AWS S3 buckets.
                        Additionally was responsible for scrubbing sensitive data from
                        repositories before publication.
                    }

                \resumeItem{Visualization/Communication of results}
                    {
                        Visualized data for researchers and produced finalized figures for publication
                        to clearly communicate patterns in data.
                        Additionally, I regularly presented results and background of my research projects
                        to a range of audiences including lab meetings, collaborative meetings, and team meetings.
                    }

                \resumeItem{Collogue consultation}
                    {
                        Provided various degrees of guidance and education to 
                        undergraduate interns, graduate students, postdocs, and various other faculty around the hutch
                        with computational tasks surrounding active research projects as well as employee onboarding.
                        This includes, but is not limited to,
                        algorithmic implementation,
                        command line interface,
                        HPC clusters,
                        batch submission (SLURM managed),
                        workflow management,
                        machine learning model exploration,
                        data formatting,
                        version control, 
                        compute environments, 
                        and build systems.
                    }

            \resumeItemListEnd
        
        % Fred Hutch Internship

        \resumeSubheading
            {Research Associate}
            {Fred Hutchinson Cancer Center}
            {Computational Biology -- Matsen Lab}
            {March 2020 - Dec. 2020, Seattle, WA}
            \resumeItemListStart

                \resumeItem{PhIP-Seq software and analysis}
                    {
                        Developed a suite of software tools for
                        analysis of raw Phage ImmunoPrecipitation Sequencing
                        (PhIP-Seq) data that consisted of;
                        a python package for modeling and computing enrichment statistics,
                        an interactive application for visualizing results of the processed data,
                        and an end-to-end (raw data $\rightarrow$ results) 
                        Nextflow pipeline for encapsulating the 
                        common analysis workflows in a portable and reproducible manner.
                        In addition to informing best-practices and future experiments carried out in wet lab,
                        This software was used to process data and produce results reported in 
                    }

                \resumeItem{Neural Networks for global epistasis modeling}
                    {
                        Worked with a graduate student to implement a global epistasis
                        modeling software package, \href{https://github.com/matsengrp/torchdms}{torchdms},
                        using the PyTorch library.
                    }

                \resumeItem{Team Organization}
                	{
                        Set up and led team meetings to brainstorm, 
                        set, short-term goals, and present results for a range
                        of team members across multiple labs and studies .
                    }

            \resumeItemListEnd

        % GE Position

        \resumeSubheading
            {Graduate Educator (BIO 410/510)}
            {University of Oregon}
            {Biology Department - Graduate Teaching Fellow}
            {Aug 2019 - March 2020, Eugene, OR}
            \resumeItemListStart

                \resumeItem{Teaching}
                    {
                        Taught and proctored exams for four classroom hours each week for 
                        a graduate level class focused around Python/Bash scripting,
                        Unix file formatting/parsing, and other algorithmic thinking activities. 
                    }

                \resumeItem{Assignment Design}
                    {
                        Constructed Docker containers and a Jupyter notebook auto-grade pipeline for 
                        students to have a unified environment for Python programming. Hosted office hours, and graded 
                        $\approx 100$ assignments/exams per week with feedback.
                    }

            \resumeItemListEnd

        % KERN / RALPH LAB

        \resumeSubheading
            {Scientific Programmer}
            {University of Oregon}
            {Institute of Ecology and Evolution -- Ralph \& Kern Co-Lab}
            {June 2018 - June 2019}
            \resumeItemListStart

                \resumeItem{Software Engineering}
                    {
                        Leveraged Software engineering best-practices of
                        architecture design, version control, continuous integration, and more
                        for various open-source software projects including
                        \href{https://tskit.dev/}{tskit}, 
                        \href{https://popsim-consortium.github.io/stdpopsim-docs/stable/index.html}{stdpopsim},
                        and \href{https://github.com/kr-colab/ReLERNN}{ReLERNN}.
                    }

                \resumeItem{Experimentation and model development}                
                    {
                        Quantified the application of
                        simulated data for training recurrent deep 
                        learning models (RNN's) to predict recombination rate estimates 
                        across the consensus genome of empirical populations.
                        This work provided the analysis pipeline used in the resulting publication \cite{Adrion_2020_relernn}.
                    }

                \resumeItem{Analysis of population demography inference tools}
                    {
                        Constructed a snakemake pipeline for manuscript analysis
                        outlining a comparison of population demography inference tools \cite{Adrion_2020_stdpopsim}.
                    }

            \resumeItemListEnd

        % RALPH / UNDERGRADUATE WORK

        \resumeSubheading
            {Undergraduate Research Assistant}
            {University of Oregon}
            {Institute of Ecology and Evolution -- Ralph \& Kern Co-Lab}
            {June 2017 - June 2018, Eugene, OR}
            \resumeItemListStart

                \resumeItem{Algorithmic implementation of tree sequence recording}
                    {
                        Implemented tree sequence recording algorithm
                        into widely used population genetic simulation software, 
                        \href{https://messerlab.org/slim/}{SLiM}, and contributed to the writing in
                        the resulting publication \cite{Haller_2019}.
                    }

                \resumeItem{Algorithmic benchmarking}
                    {
                        Utilized cluster resources,
                        visualization tools, and more to compare large scale simulations and benchmark performance across
                        a range of metrics including memory allocation and user runtime.
                        This was written up and formally presented in my
                        \href{https://www.cs.uoregon.edu/Reports/UG-201806-Galloway.pdf}{undergraduate honors thesis}.
                    }

                \resumeItem{Simulation studies of rapid evolution}
                    {
                        Developed and analyzed simulation models to study migration patterns involved in rapid evolution 
                        of stickleback fish on the Alaskan coast. 
                        I then lead the publication of this study \cite{Galloway_2020}. 
                    }

            \resumeItemListEnd
 
       % MATH GRADING

        \resumeSubheading
            {Discrete Math Course Grader}
            {University of Oregon}
            {Mathematics Department}
            {June 2016 - June 2017, Eugene, OR}
            \resumeItemListStart

                \resumeItem{Grading}
                    {
                        Graded and constructed feedback for $\approx 100$ 
                        assignments per week focused on proofs, set theory, and combinatorics.
                    }
                
            \resumeItemListEnd

    \resumeSubHeadingListEnd

%-------- Software -----------------

\section{Highlighted Software Projects}
\resumeSubHeadingListStart

    \resumeSoftwareSubheading{multidms}
        {
            \oldhref{https://matsengrp.github.io/multidms/}{\faGlobe}
            \oldhref{https://www.biorxiv.org/content/10.1101/2023.07.31.551037v1}{\faNewspaper}
            \oldhref{https://github.com/matsengrp/multidms}{\faCode}
        }
        {
            Using google's JAX library, this open-source python package 
            provides the utilities for jointly modeling multiple distinct Deep Mutational Scanning datasets.
            The interface includes a number of tools which help to easily prepare and encode data
            initialize and fit models, perform out-of-sample prediction, 
            as well as interactively visualizing 
            the resulting parameter values of interest.
            This software adds to the growing list of tools for modeling DMS data,
            and has been used to produce results for multiple studies.
        }

    \resumeSoftwareSubheading{phippery}
        {
            \oldhref{https://matsengrp.github.io/phippery/introduction.html}{\faGlobe}
            \oldhref{https://academic.oup.com/bioinformatics/article/39/10/btad583/7280694}{\faNewspaper}
            \oldhref{https://github.com/matsengrp/phippery}{\faCode}
        }
        {
            The phippery software suite provides tools for analyzing data from phage display methods 
            that use immunoprecipitation and deep sequencing to capture antibody binding to peptides, 
            often referred to as PhIP-Seq. 
            It has three main components that can be used separately or in conjunction: 
            (i) a Nextflow pipeline, phip-flow, to process raw sequencing data into a compact, 
            multidimensional dataset format and allows for end-to-end automation of reproducible workflows. 
            (ii) a Python API, phippery, which provides interfaces for tasks such as count normalization, enrichment calculation, 
            multidimensional scaling, modeling for significant enrichment of peptides, and finally 
            (iii) a Streamlit application, phip-viz, as an interactive 
            interface for visualizing the data as a heatmap in a flexible manner.
        }
    \resumeSoftwareSubheading{ReLERNN}
        {
            \oldhref{https://academic.oup.com/mbe/article/37/6/1790/5741419?login=true}{\faNewspaper}
            \oldhref{https://github.com/kr-colab/ReLERNN}{\faCode}
        }
        {
            A python-based CLI for recombination-landscape estimation 
            using recurrent neural networks (\textit{ReLERNN}).
            This software implements a deep learning method for estimating 
            a genome-wide recombination map that is accurate 
            even with small numbers of pooled or individually sequenced genomes. 
            Rather than use summaries of linkage disequilibrium as its input, 
            ReLERNN takes columns from a genotype alignment, 
            which are then modeled as a sequence across the genome 
            using a recurrent neural network.
        }

\resumeSubHeadingListEnd

%
%\item [S1.] \href{https://github.com/jgallowa07/tsencode}{TS Encode $^\dagger$} \space \space 
%        This interface takes tskit$^{S5}$ \textit{tree sequence} objects 
%        and prepares them into 3D tensors to be used for machine learning or
%        visualization of population genetic processes. 
%        
%        \space \url{https://github.com/jgallowa07/tsencode}
%
%\item [S7.] \href{https://play.google.com/store/apps/details?id=com.Nighthawks.SquirrelSuiter&hl=en_US}{Squirrel Suiter $^\dagger$} 
%        \space \space A fun 3D ``infinite flyer" I made with some friends in undergrad! 
%        

%-------- Primary Author Publications -----------------
\bibliographystyleprim{unsrt_abbrv_custom}
\nociteprim{*}
\small\bibliographyprim{first}

%-------- Supporting Author Publications -----------------
\bibliographystylesupp{unsrt_abbrv_custom}
\nocitesupp{*}
\bibliographysupp{supporting}

%-------- Professional References -----------------
\section{Professional References}

\resumeSubHeadingListStart
    
    \resumeSubItem {Dr. Erick Matsen}
        {Principal Investigator -
        %\href{https://www.fredhutch.org/en/faculty-lab-directory/matsen-frederick.html}
        %{Fred Hutchinson Cancer Research Center} -
        matsen@fredhutch.org}
\resumeSubItem {Dr. Leslie Coonrod}
        {Associate Director of Master's Program -
        %\href{https://accelerate.uoregon.edu/leslie-coonrod}
        %{Bioinformatics and Genomics Master's Program} -
        coonrod@uoregon.edu}
    \resumeSubItem {Dr. Stacey Wagner}
        {Director of Master's Program -
        %\href{https://accelerate.uoregon.edu/stacey-wagner}
        %{Director of Life Sciences Program} - 
        sdwagner@uoregon.edu}
    \resumeSubItem {Dr. Bill Cresko}
        {Research Advisor -
        %\href{https://creskolab.uoregon.edu/}
        %{Cresko Lab} - 
        wcresko@uoregon.edu}
    \resumeSubItem {Dr. Peter Ralph}
        {Principal Investigator -
        %\href{https://pages.uoregon.edu/plr/}
        %{Ralph Lab} - 
        plr@uoregon.edu}
    \resumeSubItem {Dr. Andrew Kern}
        {Principal Investigator -
        %\href{http://www.kernlab.org/} 
        %{Kern Lab} -
        adkern@uoregon.edu}
    \resumeSubItem {Dr. Ben Haller}
        {Research Advisor - 
        %\href{http://benhaller.com/}
        %{Website} - 
        haller@mac.com}
\resumeSubHeadingListEnd

%-------- Supporting Author Publications -----------------
\section{Achievements}

\resumeSubHeadingListStart
    \resumeSubItem{Graduate Thesis Project}
    {
        Machine learning for identifying synapses.
    }
    \resumeSubItem{Undergraduate Honors Thesis Project}
    {
        Speeding up the Tortoise: a case study in the optimization of forward-moving evolutionary simulations.
        The manuscript can be found on the 
        href{https://www.cs.uoregon.edu/Reports/UG-201806-Galloway.pdf}{University Arxiv}
    }
    \resumeSubItem{Game Development}
    {
        Developed a 3D infinite flyer game,
        \href{https://play.google.com/store/apps/details?id=com.Nighthawks.SquirrelSuiter&hl=en_US}{Squirrel suiter} for Windows and Android, available on Google Play.
    }
    \resumeSubItem{Community Service}
    {
        Built trail for Montana Conservation Corps and acquired 300+
        hours community service.
    }

\resumeSubHeadingListEnd

\end{document}
